src/ChannelServer/World/Inventory.cs

```diff
@@ -194,19 +204,71 @@ public Item GetItem(EquipSlot slot)
 		/// </summary>
 		/// <param name="item"></param>
 		/// <return>Index of the item.</return>
-		public int AddSilent(Item item)
+		public List<object> AddSilent(Item item)
 		{
 			var cat = item.Data.Category;
-
 			lock (_syncLock)
 			{
+				List<object> items_updated = new List<object>();
```

Comment:
Why did you make this a list of objects? If you want to return something special you should make a dedicated type for it.